src/Eye4web/ZfcUser/Settings/Factory/Mapper/DoctrineORM/UserSettingMapperFactory.php

```diff
@@ -32,7 +32,7 @@ class UserSettingMapperFactory implements FactoryInterface
      */
     public function createService(ServiceLocatorInterface $serviceLocator)
     {
-        $entityManager = $serviceLocator->get('zfcuser_doctrine_em');
+        $entityManager = $serviceLocator->get('Doctrine\ORM\EntityManager');
```

Comment:
oh, I know now, there is <code>zfcuser_doctrine_em</code> in <code>ZfcUserDoctrineORM</code>, I will add it at composer.json at require-dev to support test.